.github/workflows/build-pr.yml

```diff
@@ -10,7 +10,7 @@ on:
 jobs:
 
   build:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-22.04
```

Comment:
Can you make this change in another PR to better trace this update? 

But yes, use a "latest" version is never a good practice....